<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>waveview</class>
 <widget class="QWidget" name="waveview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>回波波形显示</string>
  </property>
  <property name="windowIcon">
   <iconset resource="styleqrc.qrc">
    <normaloff>:/icon/wave.png</normaloff>:/icon/wave.png</iconset>
  </property>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
background-image: url(:/icon/button_start.png);
background-origin: padding;
background-position: left center;
padding-left:20px;</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
background-image: url(:/icon/button_stop.png);
background-origin: padding;
background-position: left center;
padding-left:20px;</string>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>雷达扫描角度</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>固定角度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>°</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-repeat: no-repeat;
background-image: url(:/icon/button_quit.png);
background-origin: padding;
background-position: left center;
padding-left:20px;</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCustomPlot" name="wavedispaly" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>340</y>
     <width>661</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>start</zorder>
  <zorder>stop</zorder>
  <zorder>radio1</zorder>
  <zorder>radio2</zorder>
  <zorder>label_2</zorder>
  <zorder>quit</zorder>
  <zorder>wavedispaly</zorder>
  <zorder>label</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="styleqrc.qrc"/>
 </resources>
 <connections/>
</ui>
